css/style.css

```diff
@@ -0,0 +1,460 @@
+}
+
+:root {
+  --color1: #f4f5f6;
+  --color2: #c7c7c7;
+  --color3: black;
+  --main-color: #f72210;
+}
+
+body {
+  background-color: var(--color1);
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.border-temp {
+  border: 1.5px dashed blue;
+}
+
+.border-temp-1 {
+  border: 2px dashed rgb(12, 1, 1);
+}
+
+.border-temp-2 {
+  border: 3px dashed yellowgreen !important;
+}
+
+.hover-none:hover {
+  text-decoration: none;
+  color: white;
+}
+
+.hover-link-cards {
+  text-decoration-color: #f72210 !important;
+}
+
+.width-custom {
+  max-width: 1310px !important;
+}
+
+.news-catagory {
+  color: rgba(3, 3, 3, 0.4);
+  font-size: 16px;
+  font-weight: 600;
+  margin: 0.7rem 0;
+}
+
+.card {
+  border: unset;
+}
+
+.main-bg {
+  background-color: var(--main-color);
+}
+
+.main-color {
+  color: var(--main-color);
+}
+
+.h-auto {
+  height: auto !important;
+}
+
+/* Fixed navbar starts here */
+
+.fixednav {
+  position: fixed;
+  top: 0;
+  z-index: 100;
+  display: none;
+  width: 100%;
+  height: 50px;
+  background-color: #f72210;
+}
+
+.avatar img {
+  width: 20px;
+}
+
+.fixednav-btn-login {
+  color: white;
+  font-size: large;
+}
+
+.fixednav-btn-subsc {
+  background-color: #ffe400;
+  font-size: 18px;
+  border-radius: 1rem;
+  height: 30px;
+}
+
+.fixednav-btn-subsc > p {
+  font-size: 11px;
+}
+
+.fixednav-btn-toggle {
+  width: 30px;
+  height: 100%;
+}
+
+.fixednav-btn-toggle > div {
+  width: 100%;
+  height: 4px;
+  background-color: rgb(199, 194, 194);
+}
+
+/* main Header starts here */
+
+.header-top {
+  height: 95px;
+  background-color: #f72210;
+}
+
+.weather > img {
+  width: 24px;
+  height: 24px;
+}
+
+.weather p {
+  color: white;
+  font-size: 14px;
+  font-weight: 700;
+  margin: 0 0.2rem;
+}
+
+.header-top_logo > img {
+  height: 100%;
+  width: 390px;
+}
+
+.header-middle {
+  position: relative;
+  height: 53px;
+  background-color: white;
+}
+
+.header-middle::before {
+  position: absolute;
+  top: 0;
+  right: 0;
+  content: "";
+  display: block;
+  width: 70px;
+  background-color: #f72210;
+  border-left: 12px solid white;
+  border-top: 13px solid #f72210;
+}
+
+.header-middle_links {
+  width: 88%;
+}
+
+.header-middle_links li {
+  border-right: 1px solid rgb(219, 209, 209);
+}
+
+.header-middle_links a {
+  padding: 0 calc(1.7vw - 10px);
+  color: black;
+  font-family: "Roboto Condensed", sans-serif;
+  font-weight: 700;
+  font-size: 19px;
+  white-space: nowrap;
+}
+
+.header-middle_search > p {
+  font-family: "Roboto Condensed", sans-serif;
+  font-size: 19px;
+}
+
+.header-middle_search > img {
+  height: 25px;
+  width: 25px;
+}
+
+.header-bottom {
+  height: 43px;
+  background-color: white;
+  border-top: 2px #f4f5f6 solid;
+  border-bottom: 2px #f4f5f6 solid;
+}
+
+.header-bottom a {
+  margin: 0 1rem;
+  font-weight: 700;
+}
+
+.header-middle_links a:hover {
+  color: #f72210;
+  text-decoration: none;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your color: #f72210;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.red-color{
  color: #f72210;
}
```
**Html file**

```
<div class="red-color">
```
**Note:** Change with the same approach to backgrounds colors
<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
I have done so as recommended. Thank you.